tests/debugger/test_debugger_symdb.py

```diff
@@ -33,6 +33,33 @@ def _assert_symbols_uploaded(self):
                 )
 
         assert not errors, "Found system-tests-errors:\n" + "\n".join(f"- {err}" for err in errors)
+        self._assert_debugger_controller_exists()
+
+    def _assert_debugger_controller_exists(self):
+        pattern = r"[Dd]ebugger[_]?[Cc]ontroller"
+
+        def check_scope(scope):
+            name = scope.get("name", "")
+            if re.search(pattern, name):
+                scope_type = scope.get("scope_type", "")
+                if scope_type in ["CLASS", "class", "MODULE"]:
+                    return True
+
+                return False
+
+            for nested_scope in scope.get("scopes", []):
+                if check_scope(nested_scope):
+                    return True
+            return False
+
+        for symbol in self.symbols:
+            content = symbol.get("content", {})
+            if isinstance(content, dict):
+                for scope in content.get("scopes", []):
+                    if check_scope(scope):
+                        return
+
+        assert False, "No scope containing debugger controller with scope_type CLASS or MODULE was found in the symbols"
```

Comment:
A little bit cleaner

```suggestion
        raise ValueError("No scope containing debugger controller with scope_type CLASS or MODULE was found in the symbols")
```

Reply:
Oh, nice! Agree, committed.